roles/content_view_version_cleanup/tasks/delete_cv_versions.yml

```diff
@@ -0,0 +1,14 @@
+---
+- name: "delete content view versions of {{ cv_name }}"
+  theforeman.foreman.content_view_version:
```

Comment:
Should the role be collection agnostic since there are other incarnations of the collection from a naming perspective?

Reply:
given how we brand the other collection, I see no reason why the branding step can't do that for us.